examples/test.yaml

```diff
@@ -0,0 +1,23 @@
+type: prompt-engine
+config:
+  model_config:
+    max_tokens: 1024
+  description_prefix: "Hello"
+  description_postfix: "!"
```

Comment:
Same as comment above - I don't think there's really a need to have a postfix for description in chat examples